managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorContainer.java

```diff
@@ -69,18 +67,19 @@
 
     private final StampedLock rwLock = new StampedLock();
 
+    // Used to keep track of the slowest non-durable cursor.
+    private final ArrayList<Item> heapForNonDurableCursors = Lists.newArrayList();
```

Comment:
Looks if we want the track the slowest position for the non-durable cursor, it's more easier to maintain `durableCursorContainer` and `nonDurableCursorContaine`r in the ManagedLedger? so that we don't need to change the implementation of the `ManageCursorContainer`

Reply:
done